lib/board.rb

```diff
@@ -0,0 +1,82 @@
+# require_relative "./player.rb"
+# require_relative "./players/human.rb"
+# require_relative "./players/computer.rb"
+# require_relative "./game.rb"
+require 'pry'
+
+class Board
+	attr_accessor :cells
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+	def reset!
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+	def display
+		puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+		puts "-----------"
+		puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+		puts "-----------"
+		puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+	end
+
+	def position(user_input_string)
+		cells[user_input_string.chomp.to_i - 1] # user is entering 1-9.  posiitons are 0-8
+	end
+
+	def update(position_arg, player_object)
+		if valid_move?(position_arg)  #added to try and fix human#move spec
+			cells[position_arg.chomp.to_i - 1] = player_object.token
+		else
+			nil
```

Comment:
With Ruby's implicit return values, an `if` statement whose condition fails returns `nil`. So this method could be simplifed to:
```
def update(position_arg, player_object)
  if valid_move?(position_arg)
    cells[position_arg.chomp.to_i - 1] = player_object.token
  end
end
```